website/content/en/docs/building-operators/helm/quickstart.md

```diff
@@ -3,321 +3,102 @@ title: Helm Based Operator Quickstart
 linkTitle: Quickstart
 weight: 2
 ---
+## Prerequisites
 
-This guide walks through an example of building a simple nginx-operator powered by [Helm][helm-official] using tools and libraries provided by the Operator SDK.
+- [helm][helm-install] version v2+.
```

Comment:
Re: previous comment, let's simplify so users don't need to install `helm` for the quickstart.
```suggestion
```